test/parallel/test-tls-getcipher.js

```diff
@@ -25,6 +25,7 @@ server.listen(common.PORT, '127.0.0.1', function() {
   var client = tls.connect({
     host: '127.0.0.1',
     port: common.PORT,
+    ciphers: cipher_list.join(':'),
```

Comment:
Sure. Another way of fixing the test would be to use some other cipher in place of the RC4 one (and remove `ciphers`). Which approach do you think is better?